### PR TITLE
CHE-716 getting workspaces via complex key

### PR DESCRIPTION
**Pull request by mshaposhnik**
This change is [Reviewable](https://reviewable.io/reviews/codenvy/codenvy/32)

### CODE REVIEW

**Comment by garagatyi** [comment]
Reviewed 1 of 3 files at r1.
Review status: 1 of 3 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codenvy/codenvy/32)*

**Comment by garagatyi** [comment]
Reviewed 2 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codenvy/codenvy/32)*

**Comment by riuvshin** [comment]
ci-build

**Comment by riuvshin** [comment]
ci-build

**Comment by skabashnyuk** [comment]
ok

File: platform-api-client-java/core/src/main/java/com/codenvy/client/core/DefaultWorkspaceClient.java [hunk 1]
```diff
@@ -33,6 +33,12 @@
  * @author Stéphane Daviet
  */
 public class DefaultWorkspaceClient extends AbstractClient implements WorkspaceClient {
+
+    /* Composite key delimiter.
+     * See {@link WorkspaceManager#getWorkspace(String key)}
```
Comment (evoevodin) [inline platform-api-client-java/core/src/main/java/com/codenvy/client/core/DefaultWorkspaceClient.java]: `@link` won't work as long as comment is not javadoc. Use `@see WorkspaceManager#getWorkspace(String key)` instead

**Comment by evoevodin** [comment]
ok

**Pull request merged**